cli/pcluster/config/update_policy.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+from enum import Enum
+
+from pcluster import utils
+
+
+class UpdatePolicy(object):
+    """Describes the policy that rules the update of a configuration parameter."""
+
+    class CheckResult(Enum):
```

Comment:
I'm not sure this CheckResult should be part of the UpdatePolicy. 
The checkResult concept seems more related to the update action itself rather than the UpdatePolicy.

Reply:
It is strictly correlated: a CheckResult is the result of a policy check().